package.json

```diff
@@ -53,6 +54,7 @@
     "globals": "^15.9.0",
     "prettier": "^3.3.3",
     "prettier-plugin-svelte": "^3.2.6",
+    "svelte": "5.0.0-next.243",
```

Comment:
you should remove all new dev dependencies from there

I think the main issue is the missing .npmrc with the hoisted parameter so tooling will work properly

Reply:
I added svelte here so that eslint uses Svelte 5 instead of Svelte 4 but will check

Reply:
It was also missing the workspaces this is also why it was failing
There should be the same dependencies